packages/blocks/src/api/matchers.js

```diff
@@ -1,30 +1,43 @@
 /**
  * External dependencies
  */
-export { attr, prop, html, text, query } from 'hpq';
-
-/**
- * WordPress dependencies
- */
-import { create } from '@wordpress/rich-text';
+export { attr, prop, text, query } from 'hpq';
 
 /**
  * Internal dependencies
  */
 export { matcher as node } from './node';
 export { matcher as children } from './children';
 
-export function richText( selector, multilineTag ) {
+export function html( selector, multilineTag ) {
```

Comment:
Why the regular `html` matcher from `hpq` is not sufficient?

Reply:
Because this one would accept `multilineTag` as an argument for `multiline` RichText values. Otherwise we'd have to go back to the `query` matcher and map the values to and from `RichText`. I don't mind doing that with strings though. Before this was an issue because it would create a new array or object every time, but with a string we can easily compare.

Reply:
Thanks for explaining, I'm not too concerned about that. `multiline` seems fine.

Reply:
This feels a bit strange to me as well, since it's something of a preprocessing filter that doesn't serve much purpose outside the RichText use-case.

Curious: What happens if we don't have this at all?

Reply:
It would include e.g. the cite. As I said, the alternative would be to revert to using query and map the values on each render. I like that solution less, but since it's just a string now I don't care as much.

Reply:
We're looking at this too much through the lens of Rich Text. If we need the matcher to support filtering, that's one thing. But on that basis alone, "multiline" doesn't maker much sense to me. I'd rather some general support of aggregates, a property which more accurately reflects the child selector (e.g. https://github.com/WordPress/gutenberg/pull/6716#issuecomment-388907201), or reverting to query as you mention.

Reply:
Where will this discussion continue? Are we going to launch 4.0 with the `multiline` property for `html` attribute sources?

Reply:
I personally thought it was a good compromise as opposed to the "artificial" key we had to use when using the `query` matcher. And I thought this "multiline" option was supposed to fix that.